install/etcd/etcd.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   containers:
   - name: etcd
     image: openshift/origin:latest
+    imagePullPolicy: IfNotPresent
```

Comment:
@deads2k this will leave updating images to user, right? (either by removing them or pulling them manually)...

Btw. don't we want the openshift/origin image tag parametrized? also the image prefix (openshift/)